app/org/sagebionetworks/bridge/validators/StudyParticipantValidator.java

```diff
@@ -51,7 +50,7 @@ public void validate(Object object, Errors errors) {
                 errors.rejectValue("phone", "does not appear to be a phone number");
             }
             // If provided, email must be valid
-            if (isNotBlank(email) && !EMAIL_VALIDATOR.isValid(email)) {
+            if (email != null && !EMAIL_VALIDATOR.isValid(email)) {
                 errors.rejectValue("email", "does not appear to be an email address");
             }
```

Comment:
If we're treating blank emails as invalid emails rather than as no email specified, we should probably make the change further up in the "email, phone, or externalId is required" section. Either null out blank emails, or check email != null instead if isBlank(email).

Reply:
Makes sense, changed this to null pointer check only. Blanks will receive a different message (invalid email) instead.